core/services/mongodb-remote/src/test/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizerUnitTests.kt

```diff
@@ -667,7 +689,7 @@ class DataSynchronizerUnitTests {
             false
         ))
         ctx.verifyConflictHandlerCalledForActiveDoc(1, expectedLocalEvent,
-            ChangeEvent.changeEventForLocalUpdate(ctx.namespace, ctx.testDocumentId, ctx.updateDocument, ctx.testDocument, false))
+            ChangeEvent.changeEventForLocalUpdate(ctx.namespace, ctx.testDocumentId, null, ctx.testDocument, false))
```

Comment:
[q] what does a change event for a local update with a null UpdateDescription mean? is it a noop update? seems a little odd that this is something our code is producing

Reply:
ah I see now that this is produced by our mock, but I think we should just have the mock produce a real update, because a local update change event with no actual things being updated seems like something that shouldn't actually exist

Reply:
Actually I think this is due to coalescence. I can add a note here that that is what this is testing.